cli/src/lib.rs

```diff
@@ -1392,7 +1433,7 @@ fn extract_idl(cfg: &WithPath<Config>, file: &str) -> Result<Option<Idl>> {
         &*file,
         cargo.version(),
         cfg.features.seeds,
-        cfg.features.safety_checks,
+        !skip_lint && cfg.features.safety_checks,
```

Comment:
should we remove the safety_checks ffeature?

Reply:
yea it's pretty much redundant now isnt it

Reply:
yup

Reply:
I'm happy to do the work on that in another PR since I put the flag everywhere 😬

Reply:
ah just saw this. already removed it here